taca_ngi_pipeline/deliver/deliver.py

```diff
@@ -86,6 +86,11 @@ def __init__(self, projectid, sampleid, **kwargs):
         self.reportpath = getattr(self, 'reportpath', None)
         self.force = getattr(self, 'force', False)
         self.stage_only = getattr(self, 'stage_only', False)
+        # Mosler specific fields
```

Comment:
as far as I can see, these fields are only used by the `MoslerProjectDeliverer` class, so it would be better to overload that constructor and set them there instead (i.e. like you did for the `MoslerDeliverer`)